view/frontend/web/css/source/_extend.less

```diff
@@ -273,3 +273,46 @@
         }
     }
 }
+
+#payment-method-braintree-cc-vault {
+    .field {
+        &.required {
+            .label {
+                font-weight: 600;
+                &:after {
+                    content: '*';
+                    color: #e02b27;
+                    font-size: 1.2rem;
+                    margin: 0 0 0 5px;
+                }
+            }
+        }
+        .hosted-control {
+            background: #ffffff;
+            background-clip: padding-box;
+            border: 1px solid #c2c2c2;
+            border-radius: 1px;
+            font-family: 'Open Sans', 'Helvetica Neue', Helvetica, Arial, sans-serif;
+            font-size: 14px;
+            height: 32px;
+            padding: 0 9px;
+            vertical-align: baseline;
+            box-sizing: border-box;
+            position: relative;
+            width: 5rem;
```

Comment:
This will mean the width changes if the root font size is different. I wouldn't use `rem` on widths or heights unless you want them to be affected by root font size.